Server.java

```diff
@@ -22,9 +22,16 @@ void runServer(ServerUI serverUI) {
 
                 outputToClient.writeBytes("From Server: Connected...\n");
 
+                do {
+                    clientSentence = inputFromClient.readLine();
+                } while(!clientSentence.contains("Disconnect"));
```

Comment:
So this would keep waiting until the message "Disconnect is sent back to the server from the client"